src/user/user.controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Body, Controller, Get, Post } from '@nestjs/common';
+import { UserService } from './user.service';
+
+@Controller('user')
+export class UserController {
+    constructor(private userservice: UserService
+    ) {}
+
+    @Post('create')
+    createNewUser(@Body() userDto: {id:number, username: string, password: string}){
+        return this.userservice.createUser(userDto.id, userDto.username, userDto.password);
```

Comment:
Controller logic should be wrapped in try and catch phrase to handle database, io, memory and custom exceptions instead of crashing the project